akamai/cache-client.go

```diff
@@ -79,9 +81,20 @@ func NewCachePurgeClient(
 	retries int,
 	retryBackoff time.Duration,
 	log blog.Logger,
-	stats metrics.Scope,
+	stats prometheus.Registerer,
 ) (*CachePurgeClient, error) {
-	stats = stats.NewScope("CCU")
+	purgeLatency := prometheus.NewHistogram(prometheus.HistogramOpts{
+		Name:    "ccu_purge_results",
```

Comment:
The stat name is `purgeLatency` but the stat name is `ccu_purge_results`. Should it be `ccu_purge_latency` instead?